src/app/app.configuration.js

```diff
@@ -0,0 +1,20 @@
+(function () {
+    'use strict';
+
+    angular.module( 'roboconf' )
```

Comment:
Are you targetting to match our coding guidelines? If so, we don't use space inside parenthesis.